stonesoup/models/measurement/clutter.py

```diff
@@ -0,0 +1,272 @@
+# -*- coding: utf-8 -*-
+
+import numpy as np
+from scipy.stats import uniform
+from typing import Set, Union
+
+from ..base import Property
+from .base import MeasurementModel
+from ...types.detection import Clutter
+from ...types.angle import Bearing, Elevation
+from ...types.groundtruth import GroundTruthState
+from ...types.array import CovarianceMatrix
+from ...functions import cart2pol, cart2sphere
+from ...types.array import StateVector, StateVectors
+from ...types.numeric import Probability
+from ...types.state import State
+
+
+class ClutterBearingRange(MeasurementModel):
```

Comment:
I wonder if we should make these unique models, rather than `MeasurementModel` subclass?

Reply:
I'm definitely happy to do that, it wouldn't be hard since they only inherit the base `MeasurementModel`, so all I would need to do is add the `ndim_state` and `mapping` attributes. Would you like me to make that change? Thanks :)

Reply:
Following our conversation on Tuesday, I made the change. The class actually doesn't need the `ndim_state` or `mapping` attributes as it will get all relevant information from the `MeasurementModel` once it is used.